code/game/objects/effects/spiders.dm

```diff
@@ -193,8 +193,8 @@
 			S.faction = faction.Copy()
 			S.directive = directive
 			if(player_spiders)
-				S.playable_spider = TRUE
-				notify_ghosts("Spider [S.name] can be controlled", null, enter_link="<a href=?src=[REF(S)];activate=1>(Click to play)</a>", source=S, action=NOTIFY_ATTACK, ignore_key = POLL_IGNORE_SPIDER)
+				S.set_playable()
+				S.flavor_text = FLAVOR_TEXT_BLOB
```

Comment:
Why are spiderlings blob text?

Reply:
Blob text is meant to be 'antagonist with a goal' or something.